goal_src/jak1/pc/debug/anim-tester-x.gc

```diff
@@ -464,73 +464,100 @@
 
     (if (= *master-mode* 'menu)
         (anim-tester-x-interface)
-        (set! *camera-read-buttons* #t)
+        (set! *camera-read-buttons* #f)
         )
     (when (!= *master-mode* 'menu)
       (add-debug-x #t (bucket-id debug-no-zbuf) (-> self root trans) *color-white*)
       )
     (none))
 
   :code (behavior ()
+    (let
+      ((need-update (the-as symbol #f))
```

Comment:
`#f` is already a symbol, so you don't need this cast.

Reply:
Fair point. Makes sense.
I now fixed this for both games.